src/api-guides/e-receipts.markdown

```diff
@@ -1,173 +1,159 @@
+4. The redirect URI will contain an `id` and `requestToken` parameters.  
 Example:   https://{partner_redirect_URI}?id=8568a4cd-8ffc-49d6-9417-be2d69aa075f&requestToken=5l85ae5a-426f-4d6f-8af4-08648c4b696b
-5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the Concur Authorization service to obtain the official oauth2 accesstoken and refreshtoken for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
-6. Decode the **id\_token** to obtain the **sub** value and store this value as the user **id** (see [https://jwt.io](https://jwt.io))
-7. An access token is valid only for one hour.  The access token should be cached in memory and discarded after use.
-8. After the user has successfully completed the login/enrollment process, store the following with the user&#39;s profile in your database.
+5. When your application receives the redirect call strip the `id` and `requestToken` values from the URI and use those on a Post request to the SAP Concur Authorization service to obtain the official OAuth2 `accessToken` and `refreshToken` for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being geo aware](#being-geo-aware).
+6. Decode the `id\_token` to obtain the `sub` value and store this value as the user `id` (see [https://jwt.io](https://jwt.io)).
+7. An access token is valid only for one hour. The access token should be cached in memory and discarded after use.
+8. After the user has successfully completed the login/enrollment process, store the following with the user's profile in your database.
    - refresh\_token: (36 characters including dashes) valid for six months from the day and time issued.
    - refresh\_expires\_in: This is Epoch time format, convert to UTC.
    - geolocation:  to be used when making API calls on behalf of the user.
-   - Sub: (36 characters including dashes) user **id** value provided on the redirect URI.  The user **id** will be used to post receipts to the user&#39;s Concur account.
-
+   - Sub: (36 characters including dashes) user `id` value provided on the redirect URI.  The user `id` will be used to post receipts to the user's SAP Concur account.
 
-#### Web Flow - Connecting from the Partner Web / Mobile App
+### Web Flow - Connecting from the Partner Web / Mobile App
 
-1. Partner App-User clicks on **Link to Concur** button within the Partner&#39;s website/mobile App.  Partner Web/Mobile Connection Example below:
+1. Partner App-User clicks on **Link to Concur** button within the Partner's website/mobile App.  Partner Web/Mobile Connection Example below:
```

Comment:
Totally personal style: I prefer `website or mobile app` instead of `website/mobile app`.